management-service/src/main/java/pl/pwr/zpi/reports/dto/request/CreateScheduleRequest.java

```diff
@@ -0,0 +1,19 @@
+package pl.pwr.zpi.reports.dto.request;
+
+import jakarta.validation.constraints.Min;
+import lombok.Builder;
+import lombok.NonNull;
+import pl.pwr.zpi.cluster.entity.ClusterConfiguration;
+import pl.pwr.zpi.reports.dto.report.application.ApplicationConfigurationDTO;
+import pl.pwr.zpi.reports.dto.report.node.NodeConfigurationDTO;
+import pl.pwr.zpi.reports.enums.Accuracy;
+
+import java.util.List;
+@Builder
+public record CreateScheduleRequest(
+        @NonNull
+        String clusterId,
+        @NonNull
+        @Min(86400000)
```

Comment:
Can some custom error message be put here? To indicate from where does this constraint come from.

Reply:
There is already: 
{
    "errors": [
        "must be greater than or equal to 86400000"
    ]
}

Reply:
Yeah, but you don't know what must be greater and why (a domain explanation is preferred - because report generation takes up to 24h etc.)